guides/updating/update-indy-sdk-to-askar.md

```diff
@@ -0,0 +1,168 @@
+# Migrate From an Indy-sdk wallet to Aries Askar
+
+This documentation explains the process of migrating your Indy-sdk wallet to
+[Aries Askar](https://github.com/hyperledger/aries-askar).
+
+:::danger
+
+While the migration script techinically works on node.js, it is strongly
+advised not to use it, yet. The mean reason for this is that the Credential
+Definition migration is not done yet. When a credential definition is detected
+it will revert the migration process and no harm is done.
+
+:::
+
+:::caution
+
+Postgres is not supported, yet.
+
+:::
+
+:::caution
+
+The migration script is supported for 0.4.x (or moving from 0.3.x to 0.4.x).
+Please refer to [Migrating from AFJ 0.3.x to 0.4.x](./versions/0.3-to-0.4.md)
+to get to 0.4.x.
+
+:::
+
+## What does the migration do internally?
+
+The migration script does the following to make sure everything is migrated
+properly, and if anything goes wrong, it will revert back to a working state.
+
+### Create a backup
+
+Because undefined behaviour might occur, we create a backup in the new `tmp`
+directory from Aries Framework JavaScript. if some error occurs, it will be
+reverted back to the backed-up state and if no error occurs, it will delete the
+backup from the temporary directory.
+
+### Migrate the database to an Aries Askar structure
+
+The Indy-sdk and Aries Askar have different database structures. So first of
+all we need to change some table names, decrypt all the items with the old Indy
+keys, encrypt the items with the new Aries Askar keys and store them inside the
+new structure.
+
+### Try to open the wallet in the new Aries Askar structure
+
+When the wallet is correctly transformed, the wallet will be attempted to be
+openend.
+
+### Update the keys
+
+Aries Askar has a specific way to store keys and every key, defined by the
+category of `Indy::Key` will be migrated.
+
+### Update the credential definitions
+
+:::danger
+
+Migration of credential definitions is not yet supported. This is why it is
+strongly advised not to run this script in a node.js environment.
```

Comment:
We also don't transform the did entries. For peer dids this is fine, but it means you need to import the indy/sov dids you added before the dids module was available manually. We won't write any update code for this and it's something we can add to the 0.4.0 migration docs or this one

Reply:
If this script will not, and never, pick it up I think it would be better to add the docs in the 0.4.0 migration script. 

I will however add a section for updating dids and refer to that part of the 0.4.0 docs.